AppCenter/AppCenterTests/MSModelTestsUtililty.m

```diff
@@ -33,6 +36,83 @@ + (NSDictionary *)deviceDummies {
   };
 }
 
++ (NSMutableDictionary *)extensionDummies {
+  // Set up all extensions with dummy values.
+  NSDictionary *userExtDummyValues = [MSModelTestsUtililty userExtensionDummies];
+  MSUserExtension *userExt = [MSModelTestsUtililty userExtensionWithDummyValues:userExtDummyValues];
+  NSDictionary *locExtDummyValues = [MSModelTestsUtililty locExtensionDummies];
+  MSLocExtension *locExt = [MSModelTestsUtililty locExtensionWithDummyValues:locExtDummyValues];
+  NSDictionary *osExtDummyValues = [MSModelTestsUtililty osExtensionDummies];
+  MSOSExtension *osExt = [MSModelTestsUtililty osExtensionWithDummyValues:osExtDummyValues];
+  NSDictionary *appExtDummyValues = [MSModelTestsUtililty appExtensionDummies];
+  MSAppExtension *appExt = [MSModelTestsUtililty appExtensionWithDummyValues:appExtDummyValues];
+  NSDictionary *protocolExtDummyValues = [MSModelTestsUtililty protocolExtensionDummies];
+  MSProtocolExtension *protocolExt =
+  [MSModelTestsUtililty protocolExtensionWithDummyValues:protocolExtDummyValues];
```

Comment:
Second line should be tabbed over some amount... The formatter maybe have gotten this one but it's a bit annoying that it's the only multi-line assignment